src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -5061,39 +5061,40 @@ Real64 DownInterpolate4HistoryValues(Real64 OldTimeStep, Real64 NewTimeStep, std
     Real64 const newTime1 = newTime0 - NewTimeStep;
     Real64 const newTime2 = newTime1 - NewTimeStep;
     Real64 const newTime3 = newTime2 - NewTimeStep;
-    Real64 const newTime4 = newTime3 - NewTimeStep;
 
     Real64 const DSRatio = OldTimeStep / NewTimeStep; // should pretty much be an integer value 2, 3, 4, etc.
 
     if (std::abs(DSRatio - 2.0) < 0.01) { // DSRatio = 2
-        // first two points lie between oldVals[0] and oldVals[1]
+        // first two points lie between [oldVals[0] and oldVals[1])
```

Comment:
Since the first of the two values is equal to `oldVals[0]`, `oldVals[0]` is included in this range but `oldVals[1]` is not. I thought it would be useful to specify this by using the inclusive/exclusive notation in math.